assets/helpers/paymentIntegrations/paymentApi.js

```diff
@@ -149,18 +142,27 @@ function checkRegularStatus(participations: Participations, csrf: CsrfState): Ob
     }
```

Comment:
I've recently changed this response body slightly (sorry, I didn't realise we had this logic in two places) - see https://github.com/guardian/support-frontend/pull/955/files#diff-584701246b48f2e67ad18050d3dc3c3dR71

Reply:
No worries 😃

Reply:
The error message is now `failureReason`, is that it?

Reply:
Yep - although the failureReason is intended to be machine readable, whereas the 'message' was intended to be human readable (although we never used it properly!)

Reply:
It is now - thanks for the review 👍